qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -131,7 +131,11 @@ def _run_job(self, job_id, qobj, backend_options, noise_model, validate):
             self._validate(qobj, backend_options, noise_model)
         output = self._controller(self._format_qobj(qobj, backend_options, noise_model))
         end = time.time()
-        return Result.from_dict(self._format_results(job_id, output, end - start))
+        result = Result.from_dict(self._format_results(job_id, output, end - start))
+
+        self._save_job(result, backend_options)
```

Comment:
Let's generalize this a bit, because it could be interesting to leave it open for future post processing actions over results:
```python
result = self._post_process(result, backend_options)
```
The implementation of `post_process()` can be something like:
```python
def _post_process(self, result, backend_options):
  if (backend_options is not None) and ("save_jobs" in backend_options):
     self._save_job(result) 
  
  return result
  
```

Reply:
Okay, that makes sense. Will do :+1: